tests/unit/fem/test_estimator.cpp

```diff
@@ -618,7 +655,7 @@ TEST_CASE("Kelly Error Estimator on 3D NCMesh",
       {
          REQUIRE(local_errors(i) == MFEM_Approx(0.0));
       }
-      REQUIRE(estimator.GetTotalError() == MFEM_Approx(0.0));
+      // REQUIRE(estimator.GetTotalError() == MFEM_Approx(0.0));
```

Comment:
@termi-official Is it intentional that the tests on the global error are commented out now? If I uncomment these lines (658, 677, 696), the unit tests fail. I think these lines should be removed if the test is no longer appropriate. However, if we can test the TotalError here similar to 718 / 740, this would be even better. As I see it, `Kelly Error Estimator on 3D NCMesh/Perfect Approximation XYZ" no longer tests for the total error. Can you please clarify?

Reply:
I removed the tests (see my comment in the lines 275-283 in the implementation).